packages/liveblocks-core/src/room.ts

```diff
@@ -1730,12 +1723,10 @@ function makeStateMachine<
   function onClose(event: { code: number; wasClean: boolean; reason: string }) {
     context.socket = null;
 
-    clearTimeout(context.timeoutHandles.pongTimeout);
-    clearInterval(context.intervalHandles.heartbeat);
-    if (context.timeoutHandles.flush) {
-      clearTimeout(context.timeoutHandles.flush);
-    }
-    clearTimeout(context.timeoutHandles.reconnect);
+    clearTimeout(context.timers.flush);
+    clearTimeout(context.timers.reconnect);
```

Comment:
if it's a rule that all timers must are cleared on close, then maybe just

`Object.values(context.timers).forEach(clearTimeout)`; 

This way if we add more timers in the future no further code changes are needed.

Reply:
See previous comment. I think the fact they they are timers is a lousy grouping mechanism, and they should become part of different states in the eventual state machine instead.